openfisca_france/parameters/prestations/prestations_familiales/aeeh/taux_incapacite_maximal.yml

```
@@ -0,0 +1,6 @@
+taux_incapacite_maximal_aeeh:
+  reference: https://www.legifrance.gouv.fr/codes/section_lc/LEGITEXT000006073189/LEGISCTA000006156691/#LEGIARTI000006750709
+  description: Taux d'incapacité maximal au dessous duquel l'enfant peut prétendre à l'aeeh
+  values:
+    2002-04-24:
+      value: 0.8
```

Typo de retour à la ligne manquant : 🙂 
```suggestion
      value: 0.8

```